src/Categories/Category/Slice/Properties.agda

```diff
@@ -2,6 +2,9 @@
 
 open import Categories.Category
 
+-- Various properties of Slice Categories
```

Comment:
Is this changeset supposed to be part of this PR?

Reply:
It's a little sloppy to have this here, but it's just comments, so it seemed ok to throw them in.